app/manifest.yml

```diff
@@ -7,35 +7,52 @@ modules:
       title: Jenkins for Jira
       layout: basic
       icon: https://marketplace-cdn.atlassian.com/files/0c45cd96-ccf1-4490-9df4-307dded2f6e0?fileType=image&mode=full-fit
+
+    - key: get-started-page
+      resource: main
+      resolver:
+        function: redirectFromGetStarted
+      title: Jenkins for Jira
+      layout: basic
+      icon: https://marketplace-cdn.atlassian.com/files/0c45cd96-ccf1-4490-9df4-307dded2f6e0?fileType=image&mode=full-fit
```

Comment:
we re-use that icon a lot, might be time for

```
variables:
  iconUrl: https://marketplace-cdn.atlassian.com/files/0c45cd96-ccf1-4490-9df4-307dded2f6e0?fileType=image&mode=full-fit
  
...
modules:
...
      icon: ${{ variables.iconUrl }}
```

Reply:
Hahahaha you're funny. You remember this is a Forge app right where nothing is possible?